src/Exception/ExceptionInterface.php

```diff
@@ -4,11 +4,11 @@
 
 namespace Laminas\ServiceManager\Exception;
 
-use Interop\Container\Exception\ContainerException;
+use Psr\Container\ContainerExceptionInterface;
 
 /**
  * Base exception for all Laminas\ServiceManager exceptions.
  */
-interface ExceptionInterface extends ContainerException
+interface ExceptionInterface extends ContainerExceptionInterface
```

Comment:
This is potentially a BC break for anyone doing `} catch (\ Interop\Container\Exception\ContainerException) {`

Reply:
as it is aliased via `class_alias`, it is treated as the exact same Interface. So actually this should not be a BC break.

Reply:
I've added integration tests to verify that this still works:
https://github.com/laminas/laminas-servicemanager/blob/1758c7e71e7d8e51d82c0e9630b89ce03fca9c68/test/ServiceManagerContainerInteropIntegrationTest.php